### PR TITLE
adding revrec settings to gift cards

### PR DESCRIPTION
**Pull request by tilley-kyle**
Add RevRec elements (`liability_gl_account_id`, `revenue_gl_account_id`, `performance_obligation_id`) to the GiftCard model.

### CODE REVIEW

File: src/main/java/com/ning/billing/recurly/model/ShippingMethod.java
Comment (wsethbrown) [inline src/main/java/com/ning/billing/recurly/model/ShippingMethod.java:1]: Why remove the RevRec attributes in the ShippingMethod with this GiftCard PR?

Comment (tilley-kyle) [reply src/main/java/com/ning/billing/recurly/model/ShippingMethod.java:1]: Good call out, I forgot to document that. Somehow in the ShippingMethod PR, I guess when I rebased, those fields got put in twice. No idea how, should've been caught in both regular review, devqa, and my personal review. But if you check out a [couple lines above](https://github.com/tilley-kyle/recurly-java-library/pull/6/files#diff-ba7cbcb0a93f262e296fc5c3566883d48b176716d305edcb35f710cf790f869fR46-R53), they are still there.

File: src/main/java/com/ning/billing/recurly/model/ShippingMethod.java [hunk 1]
```diff
@@ -58,15 +58,6 @@ public class ShippingMethod extends RecurlyObject {
...
 
-    @XmlElement(name = "liability_gl_account_id")
-    private String liabilityGlAccountId;
-
-    @XmlElement(name = "revenue_gl_account_id")
-    private String revenueGlAccountId;
-
-    @XmlElement(name = "performance_obligation_id")
-    private String performanceObligationId;
-
```
Comment (tilley-kyle) [inline src/main/java/com/ning/billing/recurly/model/ShippingMethod.java:69]: These were somehow added in twice, so this is removing the 2nd instance of these attrs. Check up from lines 46-52 to see these are still in the ShippingMethod model.

**Review by wsethbrown — APPROVED** [review]

**Comment by wsethbrown** [comment]
DEV QA +1

**Pull request merged**